requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+mock
```

Comment:
Test requirements are in `tox.ini` and automatically installed when invoking tox. Is there a strong reason to maintain a separate file?